bpmn-model-api/generate-process-fluent-api/pom.xml

```diff
@@ -9,8 +9,8 @@
   <properties>
     <version.camunda>7.22.0</version.camunda>
     <version.junit>4.13.2</version.junit>
-    <version.h2>2.1.214</version.h2>
-    <version.slf4j>1.7.26</version.slf4j>
```

Comment:
❓ 1.7.26 is still used in the [platform parent](https://github.com/camunda/camunda-bpm-platform/blob/c759d16b0dfe682a7956c08be52de5df0d887e3e/parent/pom.xml#L48). Maybe it's better to use a consistent version? What do you think?

Reply:
I'm fine leaving the old version, too, it matches the platform. I just gave it a try and output looked the same.